pkg/operator/ceph/cluster/osd/osd.go

```diff
@@ -352,11 +352,10 @@ func (c *Cluster) startProvisioningOverNodes(config *provisionConfig) {
 
 	// no valid node is ready to run an osd
 	if len(validNodes) == 0 {
-		logger.Warningf("no valid nodes available to run an osd in namespace %s. "+
-			"Rook will not create any new OSD nodes and will skip checking for removed nodes since "+
-			"removing all OSD nodes without destroying the Rook cluster is unlikely to be intentional", c.Namespace)
+		logger.Warningf("no valid nodes available to run osds on nodes in namespace %s", c.Namespace)
 		return
```

Comment:
@travisn  Hopefully, I didn't miss this, do we still want to return here?

Reply:
Yes, this return is still intended. I was just cleaning up this message since it was misleading